pkg/sqlcache/informer/listoption_indexer_test.go

```diff
@@ -2088,3 +2099,157 @@ func TestWatchFilter(t *testing.T) {
 	}
 
 }
+
```

Comment:
This "2102" is why I broke up spun off parts of this file into `query_generator.go` .  Once this settles down I'll do that refactoring, because that PR was on the dead branch to support the `=> [...]` syntax